gum/backend-x86/gumstalker-x86.c

```diff
@@ -905,19 +969,192 @@ gum_stalker_init (GumStalker * self)
   }
 # endif
 #endif
+
+#if defined (HAVE_LINUX) && !defined (HAVE_ANDROID)
+  gum_stalker_ensure_unwind_apis_instrumented ();
+#endif
 }
 
+#if defined (HAVE_LINUX) && !defined (HAVE_ANDROID)
+
+static void
+gum_stalker_ensure_unwind_apis_instrumented (void)
+{
+  static gsize gonce_value;
+
+  if (g_once_init_enter (&gonce_value))
+  {
+    GumAttachReturn attach_ret;
+
+    gum_exec_ctx_interceptor = gum_interceptor_obtain ();
+    if (gum_exec_ctx_interceptor == NULL)
+      return;
```

Comment:
Returning here will leave the `GOnce` in a dangling state. But this function is guaranteed to not return `NULL`, so this check should be removed.